gio/src/action_entry.rs

```diff
@@ -87,6 +88,30 @@ where
         self
     }
 
+    pub fn activate_async<Fut, F>(mut self, callback: F) -> Self
+    where
+        F: Fn(&O, &SimpleAction, Option<&Variant>) -> Fut + 'static + Clone,
```

Comment:
Why require `Clone` here? I think it would be better if you would instead use an `Rc` instead of a `Box` for storing the callback. Then you get cloning for free and don't make it the user's problem.

Reply:
any chance we resolve this? it would be cool to merge this already without waiting endlessly for #902.